python/lsst/ts/externalscripts/maintel/warmup_hexapod.py

```diff
@@ -0,0 +1,94 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["WarmUpHexapod"]
+
+import yaml
+
+from lsst.ts import salobj
+
+
+class WarmUpHexapod(salobj.BaseScript):
+    """Warm up Camera/M2 Hexapod after they have been idle for a long time. 
+
+    This sequence consists of moving the hexapod (camera or m2 hexapod) in 
+    one direction until it reaches its maximum position in both positive 
+    and negative directions. It moves in discrete steps to avoid
+    a FollowingError, which that happens when performing long movements. 
+    
+    This error became more evident in April 2022 and is reported in both 
+    [FRACAS-110] and [FRACAS-111]. 
+
+    You will probably run this script a couple of times with increasing 
```

Comment:
Why not allow the Script to run with increasing step sizes? you could make `set_sizes` be an array and the default value consist of a sequence you found to be sufficient to work most of the time.

Reply:
This is done now.